packages/components/src/confirm-dialog/stories/index.js

```diff
@@ -25,6 +25,8 @@ export default {
 
 const daText = () =>
 	text( 'message', 'Would you like to privately publish the post now?' );
+const daCancelText = () => text( 'cancel button', 'No thanks' );
+const daConfirmText = () => text( 'confirm button', 'Yes please!' );
```

Comment:
Non-blocking: I'm a tiny bit concerned that we're adding more Knob stories when we have to refactor them very soon (#35665), but I'm fine if you don't have the appetite to take that on right now. Just mentioning in case you're planning on adding more stories soon.

(We'll likely also have to put the "uncontrolled" stories behind a trigger button too, because the auto-opening of the modal will be a usability issue in our docs pages.)

Reply:
Agree on both points — these could be two small follow-ups to this PR:

1. Refactor stories to Controls
2. Add a trigger to the stories with auto-opening modals

Reply:
> I'm a tiny bit concerned that we're adding more Knob stories when we have to refactor them very soon (https://github.com/WordPress/gutenberg/issues/35665)
Ah, that's really good to know, thanks - I'll bear that in mind for future Components work, and address it for this one in another PR, along with looking into the trigger button implementation!